src/Umbraco.Core/Persistence/Repositories/Implement/MemberRepository.cs

```diff
@@ -569,6 +569,36 @@ public void SetLastLogin(string username, DateTime date)
                 filterSql,
                 ordering);
         }
+        public override IEnumerable<IMember> GetPage(IQuery<IMember> query, long pageIndex, int pageSize, Ordering ordering, IQuery<IMember> filter = null)
```

Comment:
This code can be shared in one method which can accept a bool as to whether or not to include the count